### PR TITLE
docs/fix_CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by edenafek**
Fix text and some typos.

# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)?

## What does this PR do?
Fixes # (issue).

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Welcome to the PyTorch Lightning community! We're building the most advanced res
 ## Main Core Value: One less thing to remember
 
 Simplify the API as much as possible from the user perspective.
- Any additions or improvements should minimize things the user needs to remember.   
+ Any additions or improvements should minimize the things the user needs to remember.   
 
 For example: One benefit of the validation_step is that the user doesn't have to remember to set the model to .eval().
- This avoids all sorts of subtle errors the user could make.  
+ This helps users avoide all sorts of subtle errors.
```
Comment (awaelchli) [inline .github/CONTRIBUTING.md:10]:
```suggestion
 This helps users avoid all sorts of subtle errors.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -140,21 +140,21 @@ bash .run_local_tests.sh
 
 Note: if your computer does not have multi-GPU nor TPU these tests are skipped.
 
-For convenience, you can use also your own CircleCI building which will be triggered with each commit.
-This is useful if you do not test against all required dependencies version.
+For convenience, you can also use your own CircleCI building which will be triggered with each commit.
+This is useful if you do not test against all required dependencies versions.
```
Comment (awaelchli) [inline .github/CONTRIBUTING.md:144]:
```suggestion
This is useful if you do not test against all required dependency versions.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -163,15 +163,15 @@ We welcome any useful contribution! For convinece here's a recommended workflow:
...
+    We do not rely on the name convention so far you are working with your own fork. Anyway it would be nice to follow this convention `<type>/<issue-id>_<short-name>` where the types are: `bugfix`, `feature`, `docs`, `tests`, ...
 
 3. **How to rebase my PR?**
     
-    We recommend to create a PR in separate branch different from `master`, especially if you plan to submit several changes and do not want to wait until the fist one is resolved (we can work on them in parallel). Update your master with upstream (assuming you have already set [upstream](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork))
+    We recommend creating a PR in a separate branch, othen than `master`, especially if you plan on submitting several changes at once and do not want to wait until the first one is resolved (we can work on them in parallel). Update your master with upstream (assuming you have already set [upstream](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork))
```
Comment (awaelchli) [inline .github/CONTRIBUTING.md:174]:
```suggestion
    We recommend creating a PR from a separate branch other than `master`, especially if you plan on submitting several changes at once and do not want to wait until the first one is resolved (we can work on them in parallel). Update your master with upstream (assuming you have already set [upstream](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/configuring-a-remote-for-a-fork))
```

**Comment by Borda** [comment]
@williamFalcon mind check the reformulation...

**Pull request merged**